README.md

```diff
@@ -47,7 +47,7 @@ It exposes the following metrics:
 |-|-|
 | `unused_disks_count` | How many unused disks are in this provider |
 | `unused_disks_size_gb` | Total size of unused disks in this provider in GB |
-| `unused_provider_duration_ms` | How long in milliseconds took to fetch this provider information |
+| `unused_provider_duration_ms` | How long in microseconds took to fetch this provider information |
```

Comment:
This just made me realize something: `ms` is globally understood as milliseconds, but now we're using microseconds (I missed this change), so I think we should change it to `us` instead 🤔 (µs would be more correct and Prometheus support UTF-8 if I'm not mistaken but that would be too much).